plugins/lookup/supported_roles.py

```diff
@@ -77,11 +77,9 @@
     ],
     'dbt2_driver': [
         'setup_repo',
```

Comment:
i don't think setup_repo is needed for dbt2.  it just needs fuse, and sysstat, and perf.  That's all in the distro's default repos, right?

Reply:
correct, removed